CHANGELOG.md

```diff
@@ -136,6 +136,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed an issue where the model wrapper in Lite converted non-floating point tensors to float ([#10429](https://github.com/PyTorchLightning/pytorch-lightning/pull/10429))
 - Fixed an issue with inferring the dataset type in fault-tolerant training ([#10432](https://github.com/PyTorchLightning/pytorch-lightning/pull/10432))
 - Fixed dataloader workers with `persistent_workers` being deleted on every iteration ([#10434](https://github.com/PyTorchLightning/pytorch-lightning/pull/10434))
+- Fixed uploading best model checkpoint in NeptuneLogger ([#10369](https://github.com/PyTorchLightning/pytorch-lightning/pull/10369))
```

Comment:
pls move this to unreleased section

Reply:
What @Borda meant is to move it to the 1.6.0 section which is the unreleased next version.

Reply:
:facepalm: I was thinking that 1.6.0 is some "future and breaking changes" and I was looking for something like "1.5.3" etc. Ok, should be fixed in a moment.